classic-commerce.php

```diff
@@ -11,15 +11,12 @@
  *
  * @package WooCommerce
  */
-
```

Comment:
Why are all of these blank lines being removed? This shouldn't be in this PR as it's not related to these changes.

Reply:
That's probably my fault. I just copied the file code from the gist. Should I put them back?

Reply:
I restored the blank lines in commit 92a0c1d.